backend/handlers.py

```diff
@@ -722,6 +724,23 @@ def post(self):
 
   options = util.EnableCors
 
+# A handler for monitoring progress towards matching from the Presidential
+# Election Campaign Fund: http://www.fec.gov/press/bkgnd/fund.shtml
+# "... raising at least $5,000 in each of 20 states. Only contributions from
+# individuals, and only contributions up to $250, are matchable"
+class StatesHandler(webapp2.RequestHandler):
+  def get(self):
+    util.EnableCors(self)
+    totals = {}
+    for pledge in model.Pledge.all():
+      user = model.User.all().filter('email =', pledge.email).get()
+      totals[user.state] += min(pledge.amountCents, 25000) # max $250
```

Comment:
I double checked to make sure that the first $250 counts (opposed to not being allowed to count the whole donation).

Just to be clear, from [here](http://www.fec.gov/press/bkgnd/fund.shtml):

> only $250 of an individual’s contribution can be applied toward the $5,000 threshold in each state.